xml/System.Data.Common/DbDataSource.xml

```diff
@@ -126,9 +144,18 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns a <see cref="T:System.Data.Common.DbBatch" /> that's ready for execution against the <see cref="T:System.Data.Common.DbDataSource" />.</summary>
```

Comment:
The descriptions for `<Member MemberName="CreateConnection">` and `<Member MemberName="CreateDbBatch">` are switched in this file.

Fix: https://github.com/dotnet/dotnet-api-docs/pull/8319